gammapy/detect/tests/test_cwt.py

```diff
@@ -1,12 +1,109 @@
+        assert_allclose(residual.var(), 1.11539824)
+        assert_allclose(residual[100, 100], 5.)
+        assert_allclose(residual[10, 10], 0.)
+
+    def test_transform(self):
+        cwt_data = CWTData(counts=self.data['image'],
+                           background=self.data['background'],
+                           n_scale=self.kernels.n_scale)
+        self.cwt._transform(data=cwt_data)
+
+        assert_allclose(cwt_data.transform[0, 100, 100], 0.)
+        assert_allclose(cwt_data.transform[1, 100, 100], 0.)
+
+        assert_allclose(cwt_data.error[0, 100, 100], 0.0353577810278)
+        assert_allclose(cwt_data.error[0, 10, 10], 0.0135245252886)
+        assert_allclose(cwt_data.error[1, 100, 100], 0.025143605285)
+        assert_allclose(cwt_data.error[1, 10, 10], 0.0103295708553)
+
+        assert_allclose(cwt_data.approx[100, 100], 0.)
+        assert_allclose(cwt_data.approx[100, 100], 0.)
+
+        assert_allclose(cwt_data.approx_bkg[100, 100], 2.25701292944)
+        assert_allclose(cwt_data.approx_bkg[10, 10], 0.605164586712)
+
+    def test_compute_support(self):
+        cwt_data = CWTData(counts=self.data['image'],
+                           background=self.data['background'],
+                           n_scale=self.kernels.n_scale)
+        self.cwt._transform(data=cwt_data)
+        self.cwt._compute_support(data=cwt_data)
+
+        zeros = np.zeros_like(cwt_data.support, dtype=bool)
+        assert_allclose(cwt_data.support, zeros)
+
+    def test_inverse_transform(self):
+        cwt_data = CWTData(counts=self.data['image'],
+                           background=self.data['background'],
+                           n_scale=self.kernels.n_scale)
+        self.cwt._execute_iteration(data=cwt_data)
+
+        zeros = np.zeros_like(cwt_data.model)
+        assert_allclose(cwt_data.model, zeros)
+        assert_allclose(cwt_data.mysterious_result, zeros)
+
+    def test_all_cwt_iterations(self):
+        cwt_data = CWTData(counts=self.data['image'],
+                           background=self.data['background'],
+                           n_scale=self.kernels.n_scale)
+        self.cwt.analyze(data=cwt_data)
+
+        assert_allclose(cwt_data.transform[0, 100, 100], 0.)
+        assert_allclose(cwt_data.transform[1, 100, 100], 0.)
+
+        assert_allclose(cwt_data.error[0, 100, 100], 0.0353577810278)
+        assert_allclose(cwt_data.error[0, 10, 10], 0.0135245252886)
+        assert_allclose(cwt_data.error[1, 100, 100], 0.025143605285)
+        assert_allclose(cwt_data.error[1, 10, 10], 0.0103295708553)
+
+        assert_allclose(cwt_data.approx[100, 100], 0.)
+        assert_allclose(cwt_data.approx[100, 100], 0.)
+
+        assert_allclose(cwt_data.approx_bkg[100, 100], 2.25701292944)
+        assert_allclose(cwt_data.approx_bkg[10, 10], 0.605164586712)
+
+        zeros = np.zeros_like(cwt_data.support, dtype=bool)
+        assert_allclose(cwt_data.support, zeros)
```

Comment:
With first parameters `support` and `model` also were zeros arrays... I try to exchange parameters now..